### ISSUE TITLE
Attach-by-pid failing on Windows

### ISSUE THREAD

**Issue by int19h** [microsoft/ptvsd#1910]
Python 3.7 64-bit:
```
I+00000.000: Attaching to process with ID 136944

I+00000.000: Code to be injected:
             import sys;
             import codecs;
             decode = lambda s: codecs.utf_8_decode(bytearray(s))[0];
             script_path = decode([99, 58, 92, 103, 105, 116, 92, 112, 116, 118, 115, 100, 92, 115, 114, 99, 92, 112, 116, 118, 115, 100, 92, 115, 101, 114, 118, 101, 114]);
             sys.path.insert(0, script_path);
             import attach_pid_injected;
             sys.path.remove(script_path);
             host = decode([49, 50, 55, 46, 48, 46, 48, 46, 49]);
             log_dir = decode([67, 58, 47, 103, 105, 116, 47, 112, 116, 118, 115, 100, 47, 116, 101, 115, 116, 115, 47, 95, 108, 111, 103, 115, 47, 51, 46, 55, 45, 54, 52, 47, 116, 101, 115, 116, 115, 47, 112, 116, 118, 115, 100, 47, 115, 101, 114, 118, 101, 114, 47, 116, 101, 115, 116, 95, 114, 117, 110, 46, 112, 121, 47, 116, 101, 115, 116, 95, 114, 117, 110, 91, 99, 111, 100, 101, 45, 97, 116, 116, 97, 99, 104, 95, 98, 121, 95, 112, 105, 100, 93, 47, 83, 101, 115, 115, 105, 111, 110, 45, 49]) or None;
             attach_pid_injected.attach(port=61888, host=host, client=True, log_dir=log_dir)

I+00000.016: Code injector begin

E+00000.422: Traceback (most recent call last):
               File "c:\git\ptvsd\src\ptvsd/..\ptvsd\server\cli.py", line 332, in attach_to_pid
                 show_debug_info=show_debug_info_on_target_process,
               File "c:\git\ptvsd\src\ptvsd\_vendored\pydevd\pydevd_attach_to_process\add_code_to_python_process.py", line 292, in run_python_code_windows
                 print('Connecting to %s bits target' % (bits,))
             OSError: [Errno 22] Invalid argument

             Stack where logged:
               File "c:\python\3.7-64\Lib\runpy.py", line 193, in _run_module_as_main
                 "__main__", mod_spec)
               File "c:\python\3.7-64\Lib\runpy.py", line 85, in _run_code
                 exec(code, run_globals)
               File "c:\git\ptvsd\src\ptvsd\__main__.py", line 45, in <module>
                 cli.main()
               File "c:\git\ptvsd\src\ptvsd/..\ptvsd\server\cli.py", line 362, in main
                 run()
               File "c:\git\ptvsd\src\ptvsd/..\ptvsd\server\cli.py", line 335, in attach_to_pid
                 raise log.exception()
```
There's a pydevd log, but it only has one line in it, about using Cython.

The failure is consistent across all tests that use attach-by-PID, and no other tests fail. A ptvsd test run can be restricted to just those tests with:
```
tox --develop -e py37 -- -k "attach_by_pid" --ptvsd-logs
```

**Comment by fabioz** [comment]
This changed a lot in https://github.com/microsoft/ptvsd/pull/1903, so, I'll take a look at this in that pull request.

**Comment by int19h** [comment]
This appears to be fixed by #1903.
